### PR TITLE
Properties - Answers and Tests Files

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

File: answers/Properties/CorrectMultipleErrorsTest.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    class Person 
+    {
+        public $name = "Diana";
+        public $address;
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
```
Comment (TenshiLyle) [inline answers/Properties/CorrectMultipleErrorsTest.php:11]: Also this one.

Comment (cerilacanda) [reply answers/Properties/CorrectMultipleErrorsTest.php:11]: This similar to the given sample code in the q.md that we used to present to the learners. Please check this one. Thank you.

image

File: tests/Properties/CorrectMultipleErrorsTest.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+        $this->assertEquals(1, $nodes->count(), "Expecting an instantiation statement of the 'Person' class.");
+    } 
+
+    public function testEat()
+    {
+        $eat = self::$code->find('method[name="eat"]');
+
+        $this->assertEquals(1, $eat->count(), "Expecting an eat() method.");
+    }
```
Comment (TenshiLyle) [inline tests/Properties/CorrectMultipleErrorsTest.php]: Same with this one @cerilacanda

File: tests/Properties/CorrectMultipleErrorsTest.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    public function testNameProperty()
+    {
+        $name = self::$code->find('property[name="name", type="public"]');
+
+        $this->assertEquals(1, $name->count(), "Expecting a public class property named 'name'.");
+    }
```
Comment (TenshiLyle) [inline tests/Properties/CorrectMultipleErrorsTest.php]: Actually, also with properties.

**Review by TenshiLyle — CHANGES_REQUESTED** [review]

**Review by TenshiLyle — APPROVED** [review]

**Pull request merged**